### DISCUSSION TITLE
6 Atlas wallpapers, I got bored.

### DISCUSSION THREAD

**Discussion by LumiFae** [Atlas-OS/Atlas#543] [Community Artwork]
![1](https://user-images.githubusercontent.com/54886938/216160043-dcda3ff3-4a72-4cc3-832a-f4ec79aa0650.jpg)
![2](https://user-images.githubusercontent.com/54886938/216160062-ed669552-95e2-44f0-971c-661d95f92893.jpg)
![3](https://user-images.githubusercontent.com/54886938/216160070-79f128cb-8dd8-4590-8a87-f31e59779538.jpg)
![4](https://user-images.githubusercontent.com/54886938/216160082-c2e6e73a-fc3a-438d-a922-05b32ecc3f26.jpg)
![5](https://user-images.githubusercontent.com/54886938/216160085-d097a54f-feeb-4394-bcf5-d7a932806208.jpg)
![6](https://user-images.githubusercontent.com/54886938/216160086-b349e868-94f7-4bcc-9a24-fd13cf5f5d84.jpg)

**Comment by LumiFae** [comment]
p.s. all in 4k native

**Comment by ghost** [comment]
These are simply stock images with the Atlas brand slapped on it.

**Reply by LumiFae** [reply]
They aren't stock images. They are images I have used which allow modifications. Every other image uploaded to discussions is the exact same.
